lib/rust/zephyr-sys/build.rs

```diff
@@ -0,0 +1,21 @@
+use std::env;
+use std::path::PathBuf;
+
+fn main() {
+    let bindings = bindgen::Builder::default()
+        .clang_arg("-I/home/ubuntu/zephyrproject/zephyr/include")
```

Comment:
Lol. I was going to try to figure out how to get the include path out of cmake. Turns out it is rather challenging to do. We do need to figure out how to do it correctly, though.

Reply:
I did figure out how to do this, I'll push some code hopefully tomorrow.

Reply:
It seems like you were looking at an old version. Here is the best solution I could come up with for this: https://github.com/mjaun/zephyr/blob/dea56463ebbf469118ed34ee5c9e343586bf27da/cmake/modules/rust.cmake#L42